sui-execution/latest/sui-adapter/src/execution_engine.rs

```diff
@@ -903,6 +911,7 @@ mod checked {
             tx_ctx,
             gas_charger,
             advance_epoch_pt,
+            None,
```

Comment:
Would anything go wrong if we added support for tracing system transactions? In the context of Replay that should be possible, and it's something that would certainly help us debug issues during reconfiguration etc.

Reply:
I did not think that replay tool will be able to handle these but I guess everything that's executed from `execution_loop` is fair game. Will add these!